simtools/visualization/legend_handlers.py

```diff
@@ -50,6 +50,9 @@ def __init__(self):
             mst: "dodgerblue",
             sct: "black",
             sst: "darkgreen",
+            hess: "grey",
```

Comment:
Here and in the following: there is no way to have an "OtherTelescopeObject" and have them all in grey?

Reply:
I understand the wish to avoid duplication, but I think that in general it will not make a large difference. 

We have to explicitly implement MAGIC because we will actually plot MAGIC. So, we could avoid some duplication by defining "OtherTelescope" for VERITAS/HESS, but that will reduce only those two to one type and even that reduction is limited. We anyway will have to list explicitly VERITAS/HESS as keys, so in the end we will save only one function (`class HESSHandler(TelescopeHandler):`). 
Once again, I do not really think it's worth it.